src/media/css/app-list.styl

```diff
@@ -106,6 +103,31 @@ $desktop-tile-padding = 20px 50px;
     }
 }
 
+.filters-content {
+    height: 40px;
+    line-height: 40px;
+}
+
+.expand-toggle {
```

Comment:
Hmm...we could. If we move the filters sheets to their stylesheet, it'd be component-ized to .app-list-filters instead of .app-list. We could that now or later